src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -76,6 +77,28 @@ public final void testDefaultFlawless() throws IOException {
     }
 
     @Test
+    public final void testSourcePathsOneFile() throws IOException {
+        final CheckstyleAntTask antTask = getCheckstyleAntTask();
+        final FileSet examinationFileSet = new FileSet();
+        examinationFileSet.setFile(new File(getPath(FLAWLESS_INPUT)));
+        final Path sourcePath = new Path(antTask.getProject());
+        sourcePath.addFileset(examinationFileSet);
+        antTask.addPath(sourcePath);
+        antTask.execute();
```

Comment:
> Most of the ant tests just executes the task. Should I rewrite them too?

please take care of your tests only, bad examples of tests will be resolved in scope of pitest coverage task, separate issue.